packages/security-bundle/src/defs.ts

```diff
@@ -100,7 +101,10 @@ export interface ISession {
   data?: ISessionData;
 }
 
-export interface ISessionData {}
+export interface ISessionData {
+  leftSubmissionsCount?: number;
```

Comment:
Shouldn't this be stored in the XAuthBundle as it extends the data, and doesn't necessarily imply raw Security has it.